orchagent/muxorch.cpp

```diff
@@ -1154,7 +1188,9 @@ MuxOrch::MuxOrch(DBConnector *db, const std::vector<std::string> &tables,
          Orch2(db, tables, request_),
          decap_orch_(decapOrch),
          neigh_orch_(neighOrch),
-         fdb_orch_(fdbOrch)
+         fdb_orch_(fdbOrch),
+         cfgTunnelTable_(db, CFG_TUNNEL_TABLE_NAME)
```

Comment:
Why do we use CFG table here? All the attributes are available from APP_DB table, right? Also, please use the naming convention as rest of the params. i.e use `_ `seperator.

Reply:
We need to read `encap_tc_to_queue_map` and `encap_tc_color_to_dscp_map` from `TUNNEL` table.
```
"TUNNEL": {
        "MuxTunnel0": {
            "encap_tc_to_queue_map": "[TC_TO_QUEUE_MAP|AZURE_TUNNEL]",
            "encap_tc_color_to_dscp_map": "[TC_TO_DSCP_MAP|AZURE_TUNNEL]"
        }
    }
```
Yes, these attributes are also in `APP_DB`. But how can we ensure that the `TUNNEL` table is already ported into `APP_DB` when creating muxtunnel? It's `tunnelmgrd` who write the `TUNNEL` table into `APP_DB`, right?

The name of variable will be corrected. Thanks